modules/farm_fd2_school/04-vue2/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Shopping List App</title>
+    <link
+      rel="stylesheet"
+      href="main.css"
+    />
+  </head>
+
+  <body>
+    <div id="shopping-list">
+      <h1>{{ header || 'Welcome' }}</h1>
+      <form
+        v-on:submit.prevent="items.push({id: items.length + 1, label: newItem}); newItem=''; newItemHighPriority=false;"
```

Comment:
This seems to work fine to reset the content and the high priority check box. Generally when you want to do a little more work in a handler it is a good idea to encapsulate that functionality in a method.  Which is the next topic 😄